app/scripts/modules/core/pipeline/config/stages/bake/bakeStageChooseOs.component.ts

```diff
@@ -0,0 +1,41 @@
+import {module} from 'angular';
+
+export class BakeStageChooseOSController implements ng.IComponentController {
```

Comment:
I know it's not consistent in the codebase, but we're preferring to explicitly import `ng` exports, e.g.
```javascript
import {IComponentController, IComponentOptions, IScope, module} from 'angular';

export class BakeStageChooseOSController implements IComponentController {
...
```